addons/sales_team/tests/common.py

```diff
@@ -41,6 +41,11 @@ def setUpClass(cls):
             ('id', '!=', cls.sale_team.id),
         ]).action_archive()
 
+    @classmethod
+    def get_default_groups(cls):
```

Comment:
Yay another custom magic brol no-one will hear about :roll_eyes:

Reply:
Still hoping to further improve and simplify those commons (and logic) in the future, but this particular point is fine for me as it's only meant to give what was already given in the past.

All it does is give a group to the default test user (admin, or the custom one generated by the accounting common).
When you write a test, most of the time you don't want to consider access rights, you only want to test a behavior.

If you want to test access rights or test a flow with specific access rights, you should do so explicitly, with a specific user you created (or provided by the common, like the sale_user or sale_manager specified above).